components/employeeSummary/ConfirmationModal.tsx

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
I think this is fine, but this could be split into a independent component for both the Employee & Shift.